pkgs/tools/misc/calamares/default.nix

```diff
@@ -1,24 +1,26 @@
 { lib, fetchurl, boost, cmake, extra-cmake-modules, kparts, kpmcore
-, kservice, libatasmart, libxcb, libyamlcpp, parted, polkit-qt, python, qtbase
-, qtquickcontrols, qtsvg, qttools, qtwebengine, util-linux, tzdata
-, ckbcomp, xkeyboard_config, mkDerivation
+, kservice, libatasmart, libxcb, libyamlcpp, libpwquality, parted, polkit-qt, python
+, qtbase, qtquickcontrols, qtsvg, qttools, qtwebengine, util-linux, tzdata
+, ckbcomp, xkeyboard_config, mkDerivation, nixos-extensions ? false
 }:
 
 mkDerivation rec {
   pname = "calamares";
-  version = "3.2.44.3";
+  version = "3.2.49.1";
 
   # release including submodule
   src = fetchurl {
```

Comment:
Please use fetchFromGitHub.

seem to need including submodule

Reply:
Stuff under `releases/download/` are dist tarballs uploaded by upstream, not the tarballs automatically generated by GitHub (those are under `archive/`). So `fetchurl` is in fact correct here.

Reply:
Whats the difference between the source and the release tarballs?